core/build.gradle

```diff
@@ -8,6 +8,8 @@ android {
         targetSdkVersion rootProject.ext.targetSdkVersion
 
         multiDexEnabled false
+        vectorDrawables.useSupportLibrary true
+        vectorDrawables.generatedDensities = []
```

Comment:
I think this lead to problems with notification icons in the past. Could you please test this on old Android versions or research in the git history if it was there at some point and then removed?

Reply:
I can't run an emulator and I have no older Android device to test on, so I guess I'll remove it. I don't see anything in the git history either.